### DISCUSSION TITLE
希望添加“在顶栏动态选项卡下，屏蔽指定已关注用户的视频”功能

### DISCUSSION THREAD

**Discussion by DuskShadows** [the1812/Bilibili-Evolved#4103] [功能建议 / Ideas]
需求描述：希望可以有一个在顶栏动态选项卡下屏蔽指定已关注用户的视频的功能，来筛选掉不希望在顶栏动态选项卡呈现的视频动态内容。

需求来源：我关注一个类似字幕组的up主后被其大量新视频占领顶栏动态选项卡，影响观看其他up主的视频（容易错过）

补充说明：在动态页可以通过屏蔽视频类型动态解决海量新视频动态占领动态页的问题，但是顶栏动态选项卡中无法这样处理，同时也不希望取关这个up主。
需求描述：希望可以新增一个在顶栏动态选项卡下屏蔽指定已关注用户的视频的功能，来筛选掉不希望在顶栏动态选项卡呈现的视频动态内容。

**Comment by the1812** [comment]
那还是取关吧, 脚本的过滤能力会应用到顶栏, 动态, 清爽首页等各个地方, 并不能区分是哪个地方打开的
